pkg/subctl/cmd/show.go

```diff
@@ -37,6 +39,10 @@ type restConfig struct {
 	clusterName string
 }
 
+var submarinerNotInstalledError = errors.New("the server could not find the requested resource (get submariners.submariner.io submariner)")
+var submarinerUninstalledError = errors.New("submariners.submariner.io \"submariner\" not found")
```

Comment:
Why save there as Error types and then just use `.Error()` to get the string, instead of just saving the string?
In addition, why save as variables at all? They don't seem to be set anywhere, nor are they used more than once. Why not just hard code the strings?

Reply:
for easy reading

Reply:
Please inline them

Reply:
Done